client/src/App.js

```diff
@@ -23,7 +24,8 @@ function App() {
           <Route exact path="/"/>
           <Route exact path="/signup" component={SignUp} />
           <Route path="/login" component={Login} />
-		  <Route path="/profiles/:id" component={ProfileDetails} />
+		  <Route exact path="/profiles" component={ProfileListings} />
```

Comment:
If we're adding a route, we should add a link on the navbar as well, maybe "Find a sitter" for now.

Reply:
My mistake I thought it was an only front-end task